docs/DesignDocs/SGX2-EDMM-driver-interface.md

```diff
@@ -0,0 +1,296 @@
+SGX EDMM Linux Driver Interface Design
```

Comment:
Since this is linux specific, please change the filename to have "linux" in the filename